desk/app/groups-ui.hoon

```diff
@@ -288,6 +304,8 @@
   =?  suggestions  pals-running
     =+  .^(targets=(set ship) (scry %gx %pals /targets/noun))
     (~(uni in suggestions) targets)
+  =.  suggestions
+    (~(uni in suggestions) manual-contact-suggestions)
```

Comment:
Alternatively you can change the `=|  suggestions=(set ship)` from above into `=/  suggestions=(set ship)  manual-contact-suggestions`. It's fine either way.

Reply:
That feels cleaner! Switched it over